powersimdata/input/check.py

```diff
@@ -7,6 +7,7 @@
 
 # Importing the module, not anything in it, to avid a circular import
 import powersimdata.input.grid as _grid
+from powersimdata.network.europe_tub.model import TUB
```

Comment:
I'm not sure (haven't tested) if this causes the circular import issue, but regardless I wonder if this module should "know" about specific grid models. It's more of a design thing than a future concern, but suppose we had another model like the europe one which takes a `reduction` as a constructor argument. In that case we wouldn't want to hard code this module to use `TUB`. One way around this (just the first thing that popped into my head) is taking the `ModelImmutables` instance instead of the `reduction`, so we don't have to decide _here_ which grid model it would be, shifting that responsibility to the caller.

Reply:
It makes sense.

Reply:
Done. Let me know what you think.